card/mod/03_machines/lib/card/machine.rb

```diff
@@ -81,9 +81,10 @@ def self.included(host_class)
             if item.item_cards == [item]  # no pointer card
               new_input << item
             else
-              items.insert(0, item.item_cards)
+              items.insert(0, item.item_cards.reject { |ca| ca.new_card? } )
```

Comment:
Space inside parentheses detected.<br>Pass `&:new_card?` as an argument to `reject` instead of a block.